python/cppsim_wrapper.cpp

```diff
@@ -1129,6 +1137,17 @@ PYBIND11_MODULE(qulacs_core, m) {
             (void(QuantumCircuit::*)(QuantumStateBase*, UINT, UINT)) &
                 QuantumCircuit::update_quantum_state,
             py::arg("state"), py::arg("start"), py::arg("end"))
+#if 0  // not supported yet
+        .def("update_quantum_state",
+            (void (QuantumCircuit::*)(QuantumStateBase*, UINT)) &
+                QuantumCircuit::update_quantum_state,
+            "Update quantum state with random seed", py::arg("state"),
+            py::arg("seed"))
+        .def("update_quantum_state",
+            (void (QuantumCircuit::*)(QuantumStateBase*, UINT, UINT, UINT)) &
+                QuantumCircuit::update_quantum_state,
+            py::arg("state"), py::arg("start"), py::arg("end"), py::arg("seed"))
```

Comment:
We do not add multiple comments for overloaded function ( https://github.com/Qulacs-Osaka/qulacs-osaka/issues/292 ).
We use `py::overload_cast ( https://github.com/qulacs/qulacs/pull/386 ).

Reply:
I removed it with fe2a8d6